src/components/Search/Search.tsx

```diff
@@ -16,8 +18,17 @@ const Search = ({ placeholder }: SearchProps) => {
 			setSearchValue: state.setSearchValue,
 		}),
 	);
```

Comment:
To jest do usunięcia z tego miejsca i z page students oraz z `useStudentsStore`. Nic więcej tego nie używa nie musimy mnożyć kodu, który robi to samo

Reply:
Po usunięciu z page students użyj tam `useFiltersStore`

Reply:
Jest jeszcze Search dla `main page` oraz dla `requests page` do zrobienia. Wtedy możesz zmienić PR title na `feat(search)`

Reply:
up